packages/marko-web-omeda-identity-x/index.js

```diff
@@ -15,6 +16,10 @@ module.exports = (app, {
   omedaGraphQLClientProp = '$omedaGraphQLClient',
   omedaRapidIdentifyProp = '$omedaRapidIdentify',
 
+  omedaPromoCodeCookieName = 'oly_promo_src',
+  omedaPromoCodeStripQueryParam = true,
```

Comment:
Is there a specific reason the parameter would need to be stripped? I'm not sure it's necessary, since the param will be processed by our middleware and then set as a cookie